### ISSUE TITLE
Generate document failed when installing package on Ubuntu18.04

### ISSUE THREAD

**Issue by ngoan1608** [test-fullautomation/python-jsonpreprocessor#36]
Hi,

I faced the issue on generating package's document for this repo when working with build pipelines.
The error happens **only on OSD5 with Ubuntu version 18.04**, other environments works fine.

I have looked into the log but cannot find the solution (missing packages, wrong version, ...) now.
Log on console as below:
```
(/usr/local/texlive/2022/texmf-dist/tex/latex/wasysym/uwasy.fd) [1{/usr/local/t
exlive/2022/texmf-var/fonts/map/pdftex/updmap/pdftex.map}]
No file JsonPreprocessor.toc.
[1]
Chapter 1.
(./Introduction.tex
(/usr/local/texlive/2022/texmf-dist/tex/latex/psnfss/ot1pcr.fd)) [1 <./pictures
/python3-jsonpreprocessor.png>]
Chapter 2.
(./Description.tex
(/usr/local/texlive/2022/texmf-dist/tex/latex/psnfss/ts1pcr.fd) [2] [3]
[4]
! Missing number, treated as zero.
<to be read again> 
                   \protect 
l.235 ...nter}\rule{0.5\linewidth}{\linethickness}
                                                  \end{center}
? 
! Emergency stop.
<to be read again> 
                   \protect 
l.235 ...nter}\rule{0.5\linewidth}{\linethickness}
                                                  \end{center}
!  ==> Fatal error occurred, no output PDF file produced!
Transcript written on JsonPreprocessor.log.
LaTeX compiler returned 1
Error: !!! ERROR !!!
[CDocBuilder.Build] : !!! ERROR !!!
[CDocBuilder.__GenDocPDF] : LaTeX compiler not returned expected value 0!
```
Log file: [JsonPreprocessor.log](https://github.com/test-fullautomation/python-jsonpreprocessor/files/10242809/JsonPreprocessor.log)

I have tried to build many times and found that the generation of document still worked fine with commit https://github.com/test-fullautomation/python-jsonpreprocessor/commit/60da3d0cb921ccd30ba8eb3223b1dff353c76755.
Then, from commit https://github.com/test-fullautomation/python-jsonpreprocessor/commit/306e5ebe4b3b9bed0de1667804a1dd77ac0ba275, the issue comes.

Please help to verify your changes which impact to the document's generate or something is missing (related to Texlive) on OSD5 runner.

Thank you,
Ngoan

**Comment by ngoan1608** [comment]
This issue will be fixed with #39 .

**Comment by test-fullautomation** [comment]
merged #39
